fsspec/implementations/arrow.py

```diff
@@ -5,6 +5,7 @@
 import shutil
 from contextlib import suppress
 from functools import wraps
+from setuptools._vendor.packaging import version
```

Comment:
We probably shouldn't use private `_vendor` here. Looks like `from packaging import version` is the current correct way?

Reply:
Basically I would agree. But this would create a new dependency, as `packaging` is not available in python-base
While setuptools is and includes this module.
Yet - Open for discussion